aclk/agent_cloud_link.c

```diff
@@ -1117,11 +1121,17 @@ void aclk_shutdown()
 
 int aclk_init(ACLK_INIT_ACTION action)
 {
-    UNUSED(action);
-
+//    UNUSED(action);
     static int init = 0;
     int rc;
 
+    if (action == ACLK_REINIT) {
+        unsigned long delay = aclk_reconnect_delay(1);
+        info("Retrying to establish the ACLK connection in %.3f seconds", delay / 1000.0);
```

Comment:
This is not a float

Reply:
Ah actually the definition deviates.  It was  `unsigned long int`. I divide by `1000.0` to handle the conversion